samples/spring-security-hybrid-usage/k8s/deployment.yml

```diff
@@ -0,0 +1,203 @@
+---
+###  SERVICE INSTANCE CONFIGURATION  ###
+# Xsuaa instance
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: xsuaa-authn
+spec:
+  clusterServiceClassExternalName: xsuaa
+  clusterServicePlanExternalName: application
+  parameters:
+    xsappname: spring-security-hybrid-usage
+    tenant-mode: dedicated
+    oauth2-configuration:
+      credential-types:
+        - x509
+    scopes:
+      - name: "$XSAPPNAME.Read"
+        description: "Read Permissions."
+    role-templates:
+      - name: Viewer
+        description: View Data
+        scope-references:
+          - "$XSAPPNAME.Read"
+    role-collections:
+      - name: XSUAA-Viewer
+        description: Viewer (read)
+        role-template-references:
+          - "$XSAPPNAME.Viewer"
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: xsuaa-service-binding
+spec:
+  instanceRef:
+    name: xsuaa-authn
+---
+apiVersion: servicecatalog.kyma-project.io/v1alpha1
+kind: ServiceBindingUsage
+metadata:
+  name: xsuaa-service-binding-usage
+spec:
+  parameters:
+    envPrefix:
+      name: xsuaa_
+  serviceBindingRef:
+    name: xsuaa-service-binding
+  usedBy:
+    kind: deployment
+    name: spring-security-hybrid-usage
+---
+# IAS instance
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: ias-authn
+spec:
+  clusterServiceClassExternalName: identity
+  clusterServicePlanExternalName: application
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: ias-service-binding
+spec:
+  instanceRef:
+    name: ias-authn
+---
+apiVersion: servicecatalog.kyma-project.io/v1alpha1
+kind: ServiceBindingUsage
+metadata:
+  name: ias-service-binding-usage
+spec:
+  parameters:
+    envPrefix:
+      name: ias_
+  serviceBindingRef:
+    name: ias-service-binding
+  usedBy:
+    kind: deployment
+    name: spring-security-hybrid-usage
+---
+# Service manager instance
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
```

Comment:
service manager is not required here

Reply:
shouldn't we leave at least one sample with sm definition and maybe even multiple xsuaa bindings?

Reply:
mhhh... for e2e testing it would be good to have it, and hybrid sample might be the one to choose, but on the other side, i would like to keep them as simple as possible...

Reply:
I think we need to sacrifice one of the samples for now until, we don't need to use the sm anymore.